apps/db/lib/db/dataset.ex

```diff
@@ -35,6 +35,7 @@ defmodule DB.Dataset do
     field(:is_active, :boolean)
     field(:population, :integer)
     field(:nb_reuses, :integer)
+    field(:latest_data_gouv_comment_timestamp, :naive_datetime_usec)
```

Comment:
@fchabouis j'avais zappé ce point: il est préférable d'utiliser les `utc_datetime_usec` ici, car cela aura un côté "vérifiant" de la zone. La team Ecto a indiqué (https://elixirforum.com/t/why-use-utc-datetime-over-naive-datetime-for-ecto/32532/4?u=thbar) qu'ils auraient préféré faire que ça soit le cas par défaut.